src/ci/bin/test.inc.sh

```diff
@@ -110,12 +113,13 @@ cromwell::private::create_build_variables() {
         *)
             CROMWELL_BUILD_IS_CI=false
             CROMWELL_BUILD_IS_CRON="${CROMWELL_BUILD_IS_CRON-false}"
-            CROMWELL_BUILD_IS_SECURE=false
+            CROMWELL_BUILD_IS_SECURE=true
```

Comment:
👁

Reply:
If auto-vault-rendering of secrets fails during a run on one's laptop, `test.inc.sh` prints a warning and continues, letting the user know that they should be manually setting up the confs using their own secure creds. As far as the rest of the logic goes, the tests are still "secure" in this case and may continue.